### DISCUSSION TITLE
How can I simulate a right-click item

### DISCUSSION THREAD

**Discussion by minecralogy** [PrismarineJS/mineflayer#3160] [Q&A]
In 2b2t.xin, we need right-click the compass to join the queue.

If I right-click the compass again when I in the queue, I will get a message.

But if i use `bot.simpleClick.rightMouse(2)` to right-click the compass two times, i don't get a message.

**Comment by MrFiend179** [comment]
did u try
~~~js
bot.clickWindow(<Slot_Number>, 1, 0)
~~~
